### PR TITLE
New features for v0.1.1

### PR DESCRIPTION
**Pull request by duhow**

### CODE REVIEW

File: src/app.py [hunk 1]
```diff
@@ -84,29 +98,36 @@ def process_workflow_job():
...
 def github_webhook_process():
     event = request.headers.get(GithubHeaders.EVENT.value)
-    command = f"process_{event}"
 
-    if command == "process_workflow_job":
-        app.logger.debug(f"Calling function {command}")
-        process_workflow_job()
+    if event in allowed_events:
+        app.logger.debug(f"Calling function to process {event=}")
+        func = allowed_events.get(event)
```
Comment (arponpes) [inline src/app.py:129]: this is not a big deal but we already know `event` is in `allowed_events`, I would use `[]` instead of `.get()`

Comment (arponpes) [reply src/app.py:129]: also I would call it directly `allowed_events[event]()`

**Review by arponpes — DISMISSED** [review]
Nothing blocker here so I'm approving

File: src/app.py (hunk 1 above)
Comment (duhow) [reply src/app.py:129]: That will work for sure, but I'm forcing code readability in here.

Comment (arponpes) [reply src/app.py:130]: fair enough

**Review by arponpes — APPROVED** [review]

**Pull request merged**